Tests/FoundationEssentialsTests/URITemplatingTests/URITemplate_ExpressionTests.swift

```diff
@@ -0,0 +1,265 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#if canImport(FoundationEssentials)
+@testable import FoundationEssentials
+import struct FoundationEssentials.URL
+#endif
+#if FOUNDATION_FRAMEWORK
+@testable import Foundation
+import struct Foundation.URL
+#endif
+import XCTest
```

Comment:
Why use XCTest for this, but Swift Testing for the others?

Reply:
Let me convert the others. I had originally written a lot of this a long time ago, before Testing was available.